test/openssl/test_x509cert.rb

```diff
@@ -276,6 +276,18 @@ def test_marshal
     assert_equal cert.to_der, deserialized.to_der
   end
 
+  def test_load_empty
+    empty_path = Fixtures.file_path("pkey", "empty.pem")
+    certificates = OpenSSL::X509::Certificate.load(empty_path)
+    assert_equal 0, certificates.size
+  end
+
+  def test_load_fullchain
+    fullchain_path = Fixtures.file_path("pkey", "fullchain.pem")
+    certificates = OpenSSL::X509::Certificate.load(fullchain_path)
+    assert_equal 2, certificates.size
```

Comment:
Shouldn't this validate that it loaded the correct certificates, not two of the same, or `[nil, nil]`?

Reply:
Yeah it's a good idea.

Reply:
Done.